master/internal/core_experiment.go

```diff
@@ -504,7 +506,8 @@ func (m *Master) parseCreateExperiment(params *CreateExperimentParams, user *mod
 		return nil, nil, false, nil, err
 	}
 	var ok bool
-	if ok, err = project.AuthZProvider.Get().CanGetProject(*user, p); err != nil {
+	ctx := context.TODO()
```

Comment:
maybe make parseCreateExperiment take a context? 

I'm not really sure how we are going to be using this context so not really sure if a ```context.TODO()``` will cause hard to find issues